### PR TITLE
update the example code for the 0.17.0 release.

### PR DESCRIPTION
**Pull request by jkwatson**

### CODE REVIEW

File: examples/http/build.gradle [hunk 1]
```diff
@@ -7,9 +7,11 @@ ext.moduleName = "io.opentelemetry.examples.http"
...
     compile("io.opentelemetry:opentelemetry-api")
-    compile("io.opentelemetry:opentelemetry-semconv:${openTelemetryAlphaVersion}")
     compile("io.opentelemetry:opentelemetry-sdk")
-    compile("io.opentelemetry:opentelemetry-sdk-extension-autoconfigure:${openTelemetryAlphaVersion}")
     compile("io.opentelemetry:opentelemetry-exporter-logging")
+
+    //alpha modules
+    compile("io.opentelemetry:opentelemetry-semconv")
+    compile("io.opentelemetry:opentelemetry-sdk-extension-autoconfigure")
     compile "io.grpc:grpc-context"
```
Comment (bogdandrutu) [inline examples/http/build.gradle:16]: Grpc is not alpha. Maybe a new line and comment third-party?

Comment (bogdandrutu) [reply examples/http/build.gradle:16]: Also why is grpc context here?

**Review by bogdandrutu — COMMENTED** [review]
Comments apply everywhere, also inconsistent use of compile vs implementation

File: examples/http/build.gradle (hunk 1 above)
Comment (jkwatson) [reply examples/http/build.gradle:16]: great question...no clue about that! Probably an old remnant. I'll work on cleaning it up some more.

Comment (jkwatson) [reply examples/http/build.gradle]: ok, cleaned up!

**Review by bogdandrutu — APPROVED** [review]

**Pull request merged**